torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3167,6 +3167,51 @@ def sample_inputs_take_along_dim(op_info, device, dtype, requires_grad, **kwargs
             )
 
 
+def error_inputs_amax_amin(op_info, device, **kwargs):
```

Comment:
Given how close `amax_amin` error inputs and `aminmax` error inputs are, I suggest you refactor them both to call the same helper function